radio/apps/schedules/admin.py

```diff
@@ -111,11 +108,12 @@ def get_urls(self):
         return custom_urls + urls
 
     def response_change(self, request, obj):
-        msg = _('%(obj)s was changed successfully.') % {'obj': force_unicode(obj)}
+        msg = _('{obj} was changed successfully.'.format(obj=force_unicode(obj)))
```

Comment:
I prefer format over "%" syntax, but this is just personal taste. Feel free to mangle this change.